src/util/stripStyleAttribute.ts

```diff
@@ -79,7 +79,9 @@ const stripStyleAttribute = (style: string) => {
       .filter(property => property.enabled !== false)
       .find(allowedStyleProperty => property.name.startsWith(allowedStyleProperty.property))
     if (styleProperty && (!styleProperty.test || styleProperty.test(property, styles))) {
-      return acc + `${property.name}: ${property.value};`
+      return property.name === 'font-weight' && +property.value >= 500
+        ? acc + `${property.name}: 700;`
+        : acc + `${property.name}: ${property.value};`
```

Comment:
Use ternaries at the narrowest possible level to reduce redundancies. Notice here that you repeat `acc + ${property.name}` in both cases. Instead, place the ternary inside the string template.